src/queries/most-visited.sql

```diff
@@ -0,0 +1,45 @@
+--- Authorization: none
+--- Cache-Control: max-age=1296000
+--- fromDays: 30
+--- toDays: 0
+--- limit: 100
+with topurls as ( SELECT 
+    req_http_X_Owner AS owner, 
+    req_http_X_Repo AS repo,
+    CONCAT(req_http_X_Owner, req_http_X_Repo) as name,
+    req_url,
+    COUNT(time_start_usec) AS requests,
+  FROM `helix_logging_1McGRQOYFuABWBHyD8D4Ux.requests*`
+  WHERE 
+    resp_http_Content_Type LIKE "%html%" AND
```

Comment:
`like '%xyz%'` is usually "expensive" for a query. Are there so many combinations that we cannot be more specific ?

Reply:
We could drill it down to `%text/html%`, but there may be a charset appendix.